src/utils.test.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2025 Arm Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+jest.mock('path');
+import * as os from 'os';
+import * as path from 'path';
+import { getCmsisPackRootPath, isWindows } from './utils';
+
+const CMSIS_PACK_ROOT_DEFAULT = 'mock/path';
+describe('getCmsisPackRoot', () => {
+
+    beforeEach(() => {
+    });
+
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('check if CMSIS_PACK_ROOT already exists', () => {
```

Comment:
```suggestion
    it('checks if CMSIS_PACK_ROOT already exists', () => {
```

Nit: the idea of using the `it` function is to read it as part of the test description, i.e. in this case `it checks if CMSIS_PACK_ROOT already exists`.....I know....mind-blowing.....